app/views/_user_util_links.html.erb

```diff
@@ -1,6 +1,6 @@
 <div>
   <ul class="nav navbar-nav">
-    <% if render_bookmarks_control? %>
+    <% if has_user_authentication_provider? && current_or_guest_user.present? %>
```

Comment:
Yes. `render_bookmarks_control?` was moved into the controller.
https://github.com/projectblacklight/blacklight/blob/master/app/controllers/concerns/blacklight/controller.rb#L64